openpype/hosts/batchpublisher/ui/batch_publisher_model.py

```diff
@@ -0,0 +1,190 @@
+from qtpy import QtCore, QtGui
+
+
+class BatchPublisherModel(QtCore.QAbstractTableModel):
+    HEADER_LABELS = [
+        str(),
+        "Filepath",
+        "Folder",
+        "Task",
+        "Product Type",
+        "Product Name",
+        "Representation",
+        "Version",
+        "Comment"]
+    COLUMN_OF_CHECKBOX = 0
+    COLUMN_OF_DIRECTORY = 1
+    COLUMN_OF_FOLDER = 2
+    COLUMN_OF_TASK = 3
+    COLUMN_OF_PRODUCT_TYPE = 4
+    COLUMN_OF_PRODUCT_NAME = 5
+    COLUMN_OF_REPRESENTATION = 6
+    COLUMN_OF_VERSION = 7
+    COLUMN_OF_COMMENT = 8
+
+    def __init__(self, controller):
+        super(BatchPublisherModel, self).__init__()
+
+        self._controller = controller
+        self._product_items = []
+
+    def set_current_directory(self, directory):
+        self._populate_from_directory(directory)
+
+    def get_product_items(self):
+        return list(self._product_items)
+
+    def rowCount(self, parent=QtCore.QModelIndex()):
+        return len(self._product_items)
+
+    def columnCount(self, parent=QtCore.QModelIndex()):
+        return len(BatchPublisherModel.HEADER_LABELS)
```

Comment:
This should silence the hound, finally. :)

```suggestion
    def rowCount(self, parent=None):
        if parent is None:
            parent = QtCore.QModelIndex()
        return len(self._product_items)

    def columnCount(self, parent=None):
        if parent is None:
            parent = QtCore.QModelIndex()
        return len(BatchPublisherModel.HEADER_LABELS)
```

Reply:
ive implemented this and pushed the change